.dockerignore

```diff
@@ -1 +1 @@
-/release/
+#/release/
```

Comment:
was this intentional? i don't remember it - but figured i'd point it out :)

Reply:
This is a change @qrkourier made to fix the docker release workflow

Reply:
Yes, it's to workaround a bug in Docker BuildKit. They're already fixed it upstream so we'll inherit that eventually through the GitHub Action we're using.